File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/LoaiNhanVien.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyShopQuanAo
{
    public class LoaiNhanVien
    {
        public string maLoaiNhanVien { get; set; }  // primary key
        public string tenLoaiNhanVien { get; set; }

        // constructor
        public LoaiNhanVien() { }
        public LoaiNhanVien(LoaiNhanVien loaiNhanVien)
        {
            this.maLoaiNhanVien = loaiNhanVien.maLoaiNhanVien;
            this.tenLoaiNhanVien = loaiNhanVien.tenLoaiNhanVien;
        }
        public LoaiNhanVien(string maLoaiNhanVien, string tenLoaiNhanVien)
        {
            this.maLoaiNhanVien = maLoaiNhanVien;
            this.tenLoaiNhanVien = tenLoaiNhanVien;
        }
    }
}

File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/src/mypack/LoaiNhanVien.java
package mypack;

public class LoaiNhanVien {

	public String maLoaiNhanVien;	// primary key   
	public String tenLoaiNhanVien;

    // constructor
    public LoaiNhanVien() { }
    public LoaiNhanVien(LoaiNhanVien loaiNhanVien)
    {
        this.maLoaiNhanVien = loaiNhanVien.maLoaiNhanVien;
        this.tenLoaiNhanVien = loaiNhanVien.tenLoaiNhanVien;
    }
    public LoaiNhanVien(String maLoaiNhanVien, String tenLoaiNhanVien)
    {
        this.maLoaiNhanVien = maLoaiNhanVien;
        this.tenLoaiNhanVien = tenLoaiNhanVien;
    }
    
    // getter - setter
    public String getMaLoaiNhanVien() {
		return maLoaiNhanVien;
	}
	public void setMaLoaiNhanVien(String maLoaiNhanVien) {
		this.maLoaiNhanVien = maLoaiNhanVien;
	}
	public String getTenLoaiNhanVien() {
		return tenLoaiNhanVien;
	}
	public void setTenLoaiNhanVien(String tenLoaiNhanVien) {
		this.tenLoaiNhanVien = tenLoaiNhanVien;
	}
}
